laceworksdk/http_session.py

```diff
@@ -160,7 +162,10 @@ def _get_access_token(self):
             self._print_debug_response(response)
 
         except Exception:
-            raise ApiError(response)
+            if response:
+                raise ApiError(response)
+            else:
+                logger.error("Call to _get_access_token() returned no response.")
```

Comment:
Should we raise an exception no matter what?  Just raise a fancier Exception (`ApiError`) if we have a response?

```
except Exception:
    if response:
        raise ApiError(response)
    raise
```

Reply:
Yep, of course - thank you!